src/providers/oauth2.ts

```diff
@@ -186,24 +186,30 @@ export class OAuth2IdP implements IdentityProvider {
      * @param {*} callback Callback method, `function(err, authenticationData)`
      */
     public authorizeByUserPass(user: string, pass: string, callback: Callback<AuthResponse>) {
-        let scope: string[] = null;
-        if (this.authMethodConfig.endpoints.authorizeScope) {
-            scope = this.authMethodConfig.endpoints.authorizeScope.split(' ');
-        }
         const postBody = {
             grant_type: "password",
             username: user,
             password: pass,
             client_id: this.authMethodConfig.clientId,
             client_secret: this.authMethodConfig.clientSecret,
-            scope: scope,
+            scope: this.authMethodConfig.endpoints.authorizeScope
```

Comment:
You removed the check whether this is actually defined or not. Does that matter here?

Reply:
It does not matter here I think, earlier code was converting those values to arrays which is not necessary